src/app/clients/form/submitFormHelpers.ts

```diff
@@ -278,7 +182,7 @@ export const submitAddClientForm: SubmitFormHelper = async (fields, router) => {
     const clientRecord = formatClientRecord(fields);
     const ids = await insertClient(clientRecord);
     try {
-        await insertFamily([...fields.adults, ...fields.children], ids.family_id);
+        await insertFamilyMembers([...fields.adults, ...fields.children], ids.family_id);
```

Comment:
I appreciate this not your change... but I wonder if we should define a database function to do all of this operation (insert a client and insert family members) at once as a _transaction_ and call it with `supabase.rpc`. This gives us a reliable reverting functionality so there's no margin of error when it comes to error handling.

(same for editing a client)

Reply:
done!